example/src/App.js

```diff
@@ -84,6 +84,7 @@ const App = () => {
         containImage={containImage}
         activeOnHover={activeOnHover}
         navigation={navigation}
+        onMainImageClick={(id) => console.log('main image clicked', id)}
```

Comment:
I would rename it to onActiveImageClick since the selected image is named activeImage